master/internal/stream/system.go

```diff
@@ -608,41 +627,16 @@ func (ss *SubscriptionSet) UnsubscribeAll() {
 	// ss.Experiments.Subscription.Configure(nil)
 }
 
-func prepareMessageWithCache(
-	obj interface{}, cache **websocket.PreparedMessage,
-) *websocket.PreparedMessage {
-	if *cache != nil {
-		return *cache
-	}
+func prepareWebsocketMessage(obj stream.PreparableMessage) interface{} {
```

Comment:
would it make sense to move this to `socketlike.go`?

Reply:
@rb-determined-ai mentioned it should ideally live close to where the function is used, and since it's only used once I just threw that next door

Reply:
Yeah, but it's still a few hundred lines of code away from the one place it is used.  Can you put it right next to ps.Websocket()?